lib/roo_on_rails/sidekiq/loader.rb

```diff
@@ -0,0 +1,13 @@
+require 'roo_on_rails/environment'
+module RooOnRails
+  module Sidekiq
+    class Loader
+      def self.run
+        RooOnRails::Environment.load
+        ENV['RAILS_MAX_THREADS'] = (ENV.fetch('SIDEKIQ_THREADS').to_i + 1).to_s
+        ENV['DB_REAPING_FREQUENCY'] = ENV['SIDEKIQ_DB_REAPING_FREQUENCY']
+        exec 'sidekiq'
```

Comment:
I think this is good enough for this PR, but we're going to need to change this to use pg_bouncer before we'd actually be able to use it as we'd run out connections _very_ quickly otherwise. But this PR is already big enough so lets do that separately.